core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -14,9 +14,17 @@ class InstallGenerator < Rails::Generators::AppBase
     LEGACY_FRONTEND = 'solidus_frontend'
     DEFAULT_FRONTEND = 'solidus_starter_frontend'
     FRONTENDS = {
-      'none' => "#{__dir__}/frontend_templates/none.rb",
-      'solidus_frontend' => "#{__dir__}/frontend_templates/solidus_frontend.rb",
-      'solidus_starter_frontend' => "#{__dir__}/frontend_templates/solidus_starter_frontend.rb",
+      'none' => "#{__dir__}/app_templates/frontend/none.rb",
+      'solidus_frontend' => "#{__dir__}/app_templates/frontend/solidus_frontend.rb",
+      'solidus_starter_frontend' => "#{__dir__}/app_templates/frontend/solidus_starter_frontend.rb",
+    }
+
+    LEGACY_AUTHENTICATION = 'solidus_auth_devise'
```

Comment:
Is that correct?

Reply:
Good catch, I initially intended to have a pure `devise` option that would install and configure devise directly in the rails app, but ended up skipping it. This is a leftover, removing 👍